docker/php/Dockerfile

```diff
@@ -1,8 +1,15 @@
-FROM eu.gcr.io/real-estate-classifieds/shared/php:8.1 as php
+FROM mlocati/php-extension-installer:1.5 AS extension-installer
+FROM php:8.1.6-fpm-alpine3.15 AS php
+COPY --from=extension-installer /usr/bin/install-php-extensions /usr/bin/
+RUN install-php-extensions \
+      intl \
```

Comment:
grpc should be added here as well

Reply:
I know. Removed it so I didn't have to wait 20 minutes while still figuring out the github action. That's why it is also still a draft pr :smile: